TweetBook/Domain/Post.cs

```diff
@@ -17,5 +17,7 @@ public class Post
 
         [ForeignKey(nameof(UserId))]
         public IdentityUser User { get; set; }
+
+        public List<PostTag> Tags { get; set; }
```

Comment:
I think you can remember my comment about using IList such that you can afterwards cast it to whatever list type you want(array, list etc). Another used collection is IEnumerable.